pkgs/web_socket/lib/testing.dart

```diff
@@ -0,0 +1 @@
+export 'src/fake_web_socket.dart';
```

Comment:
[nit] I don't think the `FakeWebSocket` is useful outside of `fakes`. I'd add an explicit `show` here, and not expose the class.

```suggestion
export 'src/fake_web_socket.dart' show fakes;
```